packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids.ts

```diff
@@ -325,7 +325,7 @@ export const OPPORTUNITY_STANDARD_FIELD_IDS = {
   favorites: '20202020-a1c2-4500-aaae-83ba8a0e827a',
   // TODO: check if activityTargets field can be deleted
   activityTargets: '20202020-220a-42d6-8261-b2102d6eab35',
-  taskTargets: '20202020-59c0-4179-a208-4a255f04a5be',
+  taskTargets: '20202020-59c0-4279-a208-4a255f04a5be',
```

Comment:
@eliasylonen @lucasbordeau why this change?

Reply:
I don't know, I thought it was a wrong id no ?